pybamm/models/submodels/electrolyte_diffusion/constant_concentration.py

```diff
@@ -25,9 +25,8 @@ def get_fundamental_variables(self):
         c_e_n = pybamm.FullBroadcast(1, "negative electrode", "current collector")
         c_e_s = pybamm.FullBroadcast(1, "separator", "current collector")
         c_e_p = pybamm.FullBroadcast(1, "positive electrode", "current collector")
-        c_e = pybamm.Concatenation(c_e_n, c_e_s, c_e_p)
 
-        variables = self._get_standard_concentration_variables(c_e)
+        variables = self._get_standard_concentration_variables(c_e_n, c_e_s, c_e_p)
 
```

Comment:
Can get rid of `self.set_boundary_conditions` when remove function from the base model